types/socket.io/index.d.ts

```diff
@@ -820,7 +825,7 @@ declare namespace SocketIO {
         /**
          * Any query string parameters in the request url
          */
-        query: any;
+        query: { [props: string]: any };
```

Comment:
Please change it to Record to make it more readable.
I would say all query records are `string`